teamcity/pool/pool-data-source.go

```diff
@@ -0,0 +1,131 @@
+package pool
```

Comment:
May I also please ask you not to put this into a separate package, lets leave it in the flat directory structure as everything else

Reply:
Thanks! No, I agree with your idea to separate our data representation to a separate package, will also try to properly refactor the rest of the code to do the same